rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v1/IGraphConfigurationService.java

```diff
@@ -28,17 +28,18 @@
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
 
 import java.util.Set;
 
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
 /**
  * An <code>IGraphConfigurationService</code> has methods to get {@link uk.gov.gchq.gaffer.graph.Graph} configuration information
  * such as the {@link Schema} and available {@link uk.gov.gchq.gaffer.operation.Operation}s.
  */
 @Path("/graph")
-@Produces(MediaType.APPLICATION_JSON)
-@Api(value = "/graph", description = "Methods to get graph configuration information.")
+@Produces(APPLICATION_JSON)
+@Api(value = "/graph")
```

Comment:
Might be useful to have a description in the Api annotation. Same applies to the other service classes.

Reply:
The description field is deprecated in v1.5.X and isn't used anymore.